web/gds-admin-ui/public/index.html

```diff
@@ -34,6 +35,6 @@
       The build step will place the bundled scripts into the <body> tag.
```

Comment:
Note this comment - the bundled scripts will be placed into the body tag - by putting the Google scripts in the head, it will be requested before the React application.